index.html

```diff
@@ -0,0 +1,216 @@
+                the Parties of the UNFCCC (COP 27), with a view to building on previous successes and paving the way for
+                future ambition to effectively tackle the global challenge of climate change. </p>
+            <p class=" date-about">Please Contact us per Email for any further questions
+                about Sharm el-Sheikh Climate Change Conference
+            </p>
+            <p class=" place-about">info@cop27.eg</p>
+        </article>
+    </section>
+    <section>
+        <h1 class="h1 h1-about">creative commons
+            Sharm el-Sheikh Climate Change Conference 2022 Logo
+        </h1>
+        <hr class="hr-about-2">
+        <p class="articl-2">The globe incorporated in the logo represents how we should contain earth and keep it
+            preserved, balanced and safe. It also represents Africa, the continent hosting COP27 in 2022. </p>
+        <img class="art-logo" src="img/logo1.png" alt="cop-27-logo">
+        <h1 class="h1 h1-about">See the past climate change confrence</h1>
+        <hr class="hr-about-4">
+        <p class="articl-2">Take a look at the last climate change conference Lorem ipsum dolor sit amet Lorem, ipsum
+            dolor.</p>
+        <div class="img-big-container">
+            <div class="img-container">
+                <img class="cop26-1" src="img/cop-2-cop26-2.png" alt="cop-26 photo">
+                <div class="center">United Nations climate summit UK</div>
+            </div>
+            <div class="img-container">
+                <img class="cop26-1" src="img/cop-1-26-1-960x640.png" alt="cop-26 photo">
+                <div class="center">United Nations climate summit Madrid</div>
+            </div>
+        </div>
+    </section>
+    <footer>
+        <h3 class="footer-h">Partner</h3>
+        <hr class="hr-about">
+        <div class="footer-container">
+            <img class="footer-img" src="img/air.png" alt="un">
+            <img class="footer-img" src="img/airbnb.png" alt="airbnb">
+            <img class="footer-img" src="img/daumkakao.PNG" alt="daumkakao">
+            <img class="footer-img" src="img/google.png" alt="google">
+            <img class="footer-img" src="img/mozilla.png" alt="mozilla">
+            <img class="footer-img" src="img/naver.PNG" alt="naver">
+            <img class="footer-img" src="img/egy.png" alt="Egypt air logo">
+        </div>
+        <div class="white">
+            <div>
+                <img src="img/microverse.png" width="60px" alt="microverse-logo">
+                <img src="img/logo.png" alt="summit-logo">
+            </div>
+            <p>All rights reserved
+                <span>&copy</span>Made with love <span class="microverse">&hearts;</span>
+                Original design by <a href="https://www.behance.net/adagio07">Cindy Shin</a>
+                developed by Microverse student Mostafa Shahat.
+            </p>
+        </div>
+    </footer>
```

Comment:
- [ ] Please add the about section to its own `about.html` file. The home and about pages are required to be on two separate pages as mentioned in the requirements.